tests/serve/test_sglang.py

```diff
@@ -49,8 +49,9 @@ class SGLangConfig(EngineConfig):
         env={},
         models_port=8000,
         request_payloads=[
-            chat_payload_default(),
```

Comment:
why not update chat_payload_default function to have a [default](https://github.com/ai-dynamo/dynamo/blob/1a9aeab4b76a56d222eb418cb7361dc5ab3031dc/tests/utils/payload_builder.py#L23) temperature=0.0 in one spot instead of changing it at all the call sites?